webapp/src/css/enketo/medic.less

```diff
@@ -434,8 +445,7 @@
     .option-wrapper > label:not(.filler) {
       padding-top: 9px;
       padding-bottom: 9px;
-      margin-top: 1px;
-      margin-bottom: 1px;
+      margin: 0;
```

Comment:
You've set `margin: 0` on the desktop version so we don't need to overwrite it in the mobile version. I think you can just drop this line altogether.

```suggestion
```